README.md

```diff
@@ -10,30 +10,39 @@ This is the official [cluster-api](https://github.com/kubernetes-sigs/cluster-ap
 
 ![Packetbot works hard to keep Kubernetes cluster in a good shape](./docs/banner.png)
 
+## Upgrading to v0.8.X
```

Comment:
We could use each heading to call out the upstream changes that have been adopted (and therefor, upstream requirements apply).

```md
-  Now based on CAPI 1.6 - see upstream documentation for migration notes and details <https://github.com/kubernetes-sigs/cluster-api/releases/tag/v1.6.0>
```

(or a better link geared toward that information)


We could just bundle that notice in our GitHub release if we don't include it in the README.md